test/suites/smoke/test-state-v1-migration.ts

```diff
@@ -0,0 +1,27 @@
+import "@moonbeam-network/api-augment/moonbase";
+import { ApiPromise } from "@polkadot/api";
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+
+describeSuite({
+  id: "S27",
+  title: `State V1 Migration status should not be in an error state`,
+  foundationMethods: "read_only",
+  testCases: ({ context, it, log }) => {
+    let paraApi: ApiPromise;
+
+    beforeAll(async function () {
+      paraApi = context.polkadotJs("para");
+    });
+
+    it({
+      id: "C100",
+      title: "Migration status should not be in an error state",
```

Comment:
The test title does not match with its behaviour. In the test we are checking that `error` is present in `stateMigrationStatusValue`.

Reply:
Good catch, I had it reversed.